.github/actions/assign-triage-role/assign_triage_role/templates/pr_template.md

```diff
@@ -0,0 +1,66 @@
+<!--
+Thank you for contributing! Please be sure to read our contribution guidelines: https://github.com/apache/trafficcontrol/blob/master/CONTRIBUTING.md
+If this closes or relates to an existing issue, please reference it using one of the following:
+
+Closes: #ISSUE
+Related: #ISSUE
+
+If this PR fixes a security vulnerability, DO NOT submit! Instead, contact
+the Apache Traffic Control Security Team at security@trafficcontrol.apache.org and follow the
+guidelines at https://apache.org/security regarding vulnerability disclosure.
+-->
```

Comment:
If I ever have to edit this template, the first thing I'm going to do is get rid of all these comments meant for human authors on GitHub. They aren't hurting anything, but I'm just saying it'd increase the signal-to-noise ratio in this file.

Reply:
Removed human-intended comments in acc658fce1